pkg/kube/inject/webhook.go

```diff
@@ -204,7 +208,7 @@ func NewWebhook(p WebhookParameters) (*Webhook, error) {
 
 	if p.KubeClient != nil {
```

Comment:
Do we still need this check?

Reply:
I don't think this PR changes this. I think it can still be nil in (very niche) cases where we don't have a k8s connection at all

Reply:
This can be removed in another pr